samples/bluetooth/direct_test_mode/Kconfig

```diff
@@ -66,6 +66,13 @@ config DTM_POWER_CONTROL_AUTOMATIC
 	  possible value. If this option is disabled, user can set the SoC output power and the
 	  front-end module gain with the separate vendor specific commands.
 
+config DTM_FAST_RAMP_UP
+	bool "Enable radio fast ramp up mode"
+	default n
```

Comment:
Why not use it as default? I checked with @ppryga-nordic, that SoftDevice uses it by default, so maybe we should align the DTM to the SD?